paddle配置IOS移动端时出现的配置问题

您好，我编译的了IOS版的paddle(SIMULATOR,x86_64)，所用命令如下。
cmake -DCMAKE_SYSTEM_NAME=iOS \
      -DIOS_PLATFORM=SIMULATOR \
      -DIOS_ARCH="x86_64" \
      -DIOS_USE_VECLIB_FOR_BLAS=ON \
      -DCMAKE_INSTALL_PREFIX=your/path/to/install \
      -DWITH_C_API=ON \
      -DWITH_TESTING=OFF \
      -DWITH_SWIG_PY=OFF \
      ..

但是在按照 https://github.com/PaddlePaddle/Mobile/pull/26 中的指导配置完后，遇到如下问题：

d: warning: ignoring file /Users/yukang/Desktop/code/Paddle-iOS-Demo/Paddle-iOS-Demo/third_party/libjpeg.a, missing required architecture x86_64 in file /Users/yukang/Desktop/code/Paddle-iOS-Demo/Paddle-iOS-Demo/third_party/libjpeg.a (2 slices)
Undefined symbols for architecture x86_64:
  "paddle::simd::internal::batchAddToImpl(float*, float const**, int, unsigned long)", referenced from:
      void paddle::CpuMatrix::mul<paddle::CacheRowCpuMatrix, paddle::CpuMatrix>(paddle::CpuSparseMatrix*, paddle::CacheRowCpuMatrix*, paddle::CpuMatrix*, float, float) in libpaddle_capi_whole.a(Matrix.cpp.o)
      void paddle::CpuMatrix::mul<paddle::SparseRowCpuMatrix, paddle::CpuMatrix>(paddle::CpuSparseMatrix*, paddle::SparseRowCpuMatrix*, paddle::CpuMatrix*, float, float) in libpaddle_capi_whole.a(Matrix.cpp.o)
      void paddle::CpuMatrix::mul<paddle::CpuMatrix, paddle::CpuMatrix>(paddle::CpuSparseMatrix*, paddle::CpuMatrix*, paddle::CpuMatrix*, float, float) in libpaddle_capi_whole.a(Matrix.cpp.o)
  "paddle::simd::internal::addToImpl(float*, float const*, unsigned long)", referenced from:
      paddle::CpuMatrix::addBias(paddle::Matrix&, float) in libpaddle_capi_whole.a(Matrix.cpp.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)


请问有什么办法能帮帮我吗？

--- a/paddle/math/CMakeLists.txt
+++ b/paddle/math/CMakeLists.txt
@@ -26,8 +26,6 @@ else()
 endif()
 
 if(MOBILE_INFERENCE)
-    list(REMOVE_ITEM MATH_SOURCES
-         ${CMAKE_CURRENT_SOURCE_DIR}/SIMDFunctions.cpp)
     # Remove sparse
     list(REMOVE_ITEM MATH_HEADERS
          ${CMAKE_CURRENT_SOURCE_DIR}/CpuSparseMatrix.h

--- a/paddle/math/SIMDFunctions.h
+++ b/paddle/math/SIMDFunctions.h
@@ -116,9 +116,11 @@ inline bool vec_check(size_t len) {
 }
 
 namespace internal {
+#ifdef __SSE3__
 void addToImpl(float* a, const float* b, size_t len);
 void batchAddToImpl(float* a, const float* b[], int batch, size_t len);
 void colMaxImpl(float* result, const float* data, int dim, int numSamples);
+#endif
 #ifdef __AVX__
 void decayL1AvxImpl(float* dst, float* src, float lambda, size_t len);
 void decayL1AvxImpl(

